src/api/index.ts

```diff
@@ -187,3 +187,21 @@ export async function getRecentBlocks(
   }
   return blocks;
 }
+
+// TODO(jill): update this function name once in mainnet
+export async function getStake(
+  nodeUrl: string,
+  delegatorAddress: Types.Address,
+): Promise<Types.MoveValue[]> {
+  const client = new AptosClient(nodeUrl);
+  const payload: Types.ViewRequest = {
+    function:
+      "0x1310dc820487f24755e6e06747f6582118597a48868e2a98260fa8c3ee945cbd::delegation_pool::get_stake",
```

Comment:
Can we define the address here as a constant somewhere? This would allow changing it to 0x1 more easily later on.